src/test/java/net/phptravels/WebLogin.java

```diff
@@ -1,51 +1,70 @@
 package net.phptravels;
 
 import com.qaprosoft.carina.core.foundation.IAbstractTest;
-import com.qaprosoft.carina.core.foundation.utils.ownership.MethodOwner;
-import com.qaprosoft.carina.core.foundation.utils.tag.Priority;
-import com.qaprosoft.carina.core.foundation.utils.tag.TestPriority;
 import net.phptravels.gui.pages.*;
-import com.zebrunner.agent.core.annotation.TestLabel;
 import org.testng.Assert;
 import org.testng.annotations.Test;
 
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.util.Properties;
 
-public class WebLogin implements IAbstractTest {
 
+public class WebLogin implements IAbstractTest {
+    Properties properties = new Properties();
+    {
+        try {
```

Comment:
you don't have to create your one file reading methods, Carina already have it. when you added something to testdata.properties you can read data using `R.TESTDATA.get("test_credentials")`